examples/randomized-benchmark/two-qubit-rb/readme.md

```diff
@@ -0,0 +1,86 @@
+# Randomized Benchmarking for two-qubit gates using baking
+
+Randomized benchmarking (RB) is a procedure to generate an 
+average figure-of-merit characterizing the fidelity of 
+qubit operations. This procedure is useful as full process tomography 
+may be prohibitively costly in terms of the number of required operations.   
+RB utilizes operations from the Clifford group, which in the case of two qubits contains 11520 elements
+(for $$n$$ qubits $$
+\left|\mathcal{C}_{n}\right|=2^{n^{2}+2 n} \prod_{j=1}^{n}\left(4^{j}-1\right)
+$$).
+
+We propose to illustrate the use of baking tool to perform this protocol.
+
+## The class *RBTwoQubits*
+
+The pattern behind RB is always the same: 
+- Generate a random sequence composed of Cliffords
+- Append to the sequence one Clifford reverting the action of the random sequence, such that the identity circuit is played
+- Measure the survival probability
+- Repeat prior steps for varying number of Cliffords within the sequence
+
+In order to have an easy way to derive all Cliffords of the two qubit Clifford group, 
+we use the decomposition illustrated in the following reference: https://arxiv.org/pdf/1210.7011.pdf.
+This decomposition allows an easy derivation of the group from the Clifford group for one qubit, and uses three main two qubit gates
+(that shall be decomposed later into a native set of gates by the user depending on its own hardware):
+- CNOT
+- SWAP
```

Comment:
who is playing SWAP? As far as I know there is no architecutre that can directly implement SWAP, if there is I'd be happy to know

Reply:
Assuming no one can, the idea is that those gates are the ones allowing the easy class decomposition depicted in the reference given. That is, regardless of two qubit native gates that the user wants to implement, the selection of the Clifford will always be done w.r.t to the class type defined by the presence of a CNOT/SWAP/iSWAP in the circuit. For this reason and to make the code hardware agnostic, we ask the user to provide a macro for each of those gates (if it is native to the hardware, it's the best but if it's not, he will simply have to play a bunch of single qubit gates as well to pass from one two qubit gate to another according to appropriate circuit identities). 
I will explain it better here